src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -203,32 +160,32 @@ function OptionRowLHN(props) {
                             styles.sidebarLink,
                             styles.sidebarLinkInnerLHN,
                             StyleUtils.getBackgroundColorStyle(theme.sidebar),
-                            props.isFocused ? styles.sidebarLinkActive : null,
-                            (hovered || isContextMenuActive) && !props.isFocused ? props.hoverStyle || styles.sidebarLinkHover : null,
+                            isFocused ? styles.sidebarLinkActive : null,
+                            (hovered || isContextMenuActive) && !isFocused ? styles.sidebarLinkHover : null,
                         ]}
                         role={CONST.ROLE.BUTTON}
                         accessibilityLabel={translate('accessibilityHints.navigatesToChat')}
-                        needsOffscreenAlphaCompositing={props.optionItem.icons.length >= 2}
+                        needsOffscreenAlphaCompositing={(optionItem?.icons?.length ?? 0) >= 2}
                     >
                         <View style={sidebarInnerRowStyle}>
                             <View style={[styles.flexRow, styles.alignItemsCenter]}>
-                                {!_.isEmpty(optionItem.icons) &&
+                                {optionItem.icons?.length &&
                                     (optionItem.shouldShowSubscript ? (
                                         <SubscriptAvatar
-                                            backgroundColor={hovered && !props.isFocused ? hoveredBackgroundColor : subscriptAvatarBorderColor}
-                                            mainAvatar={optionItem.icons[0]}
-                                            secondaryAvatar={optionItem.icons[1]}
-                                            size={props.viewMode === CONST.OPTION_MODE.COMPACT ? CONST.AVATAR_SIZE.SMALL : CONST.AVATAR_SIZE.DEFAULT}
+                                            backgroundColor={hovered && !isFocused ? hoveredBackgroundColor : subscriptAvatarBorderColor}
+                                            mainAvatar={optionItem?.icons?.[0]}
+                                            secondaryAvatar={optionItem?.icons?.[1]}
```

Comment:
You check several lines above that optionItem.icons are exist
```suggestion
                                            mainAvatar={optionItem.icons[0]}
                                            secondaryAvatar={optionItem.icons[1]}
```